src/lib.rs

```diff
@@ -245,14 +245,64 @@ macro_rules! fuzz_target {
         };
     };
 
+    (|mut $bytes:ident| $body:expr) => {
```

Comment:
I don't think you need to duplicate an entire branch. I feel like it to change the `$bytes` to match for `pat_param` or something.

Reply:
Thanks for the tip, I will check it out. I am not experienced in working with macros.